typegraphql-prisma-crud has an error

😡😡😡

```
$ npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: graphql-typegraphql-crud@1.0.0
npm ERR! Found: @prisma/client@4.0.0
npm ERR! node_modules/@prisma/client
npm ERR!   @prisma/client@"4.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @prisma/client@"~4.4.0" from typegraphql-prisma@0.22.0
npm ERR! node_modules/typegraphql-prisma
npm ERR!   dev typegraphql-prisma@"0.22.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/nikolasburk/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/nikolasburk/.npm/_logs/2022-10-25T10_19_06_026Z-debug.log
```

--- a/.github/readmes/typescript/graphql-typegraphql-crud/README.md
+++ b/.github/readmes/typescript/graphql-typegraphql-crud/README.md
@@ -6,7 +6,22 @@ __INLINE(../_setup-0.md)__
 curl https://codeload.github.com/prisma/prisma-examples/tar.gz/latest | tar -xz --strip=2 prisma-examples-latest/typescript/graphql-typegraphql-crud
 __INLINE(../_setup-1.md)__
 cd graphql-typegraphql-crud
-__INLINE(../_setup-2.md)__
+npm install --legacy-peer-deps
+```
+
+> Note: The `--legacy-peer-deps` flag installs the Prisma version that is compatible with `typegraphql-prisma` as a peer dependency. Refer to [`typegraphql-prisma`'s releases](https://github.com/MichalLytek/typegraphql-prisma/releases) to check the Prisma version that's compatible with the latest `typegraphql-prisma` version.
+
+<details><summary><strong>Alternative:</strong> Clone the entire repo</summary>
+
+Clone this repository:
+
+```
+git clone git@github.com:prisma/prisma-examples.git --depth=1
+```
+
+Install npm dependencies:
+
+```
 cd prisma-examples/typescript/graphql-typegraphql-crud
 __INLINE(../_setup-3.md)__
 

--- a/typescript/graphql-typegraphql-crud/README.md
+++ b/typescript/graphql-typegraphql-crud/README.md
@@ -16,8 +16,10 @@ Install npm dependencies:
 
 ```
 cd graphql-typegraphql-crud
-npm install
+npm install --legacy-peer-deps
 ```
+
+> Note: The `--legacy-peer-deps` flag installs the Prisma version that is compatible with `typegraphql-prisma` as a peer dependency. Refer to [`typegraphql-prisma`'s releases](https://github.com/MichalLytek/typegraphql-prisma/releases) to check the Prisma version that's compatible with the latest `typegraphql-prisma` version.
 
 <details><summary><strong>Alternative:</strong> Clone the entire repo</summary>
 

